cl/corpus_importer/management/commands/ready_mix_cases_project.py

```diff
@@ -81,13 +87,56 @@ def get_and_store_starting_case_ids(options: OptionsType, r: Redis) -> None:
     logger.info(f"Finished setting starting pacer_case_ids.")
 
 
-def query_results_in_es(r):
+def query_results_in_es(options):
     """Query results in ES.
-    :param r: The Redis DB to connect to as a connection interface or str that
-    can be handed off to make_redis_interface.
+    :param options: The options from the handle method
     :return None
     """
-    pass
+
+    court_ids = get_bankruptcy_courts(["all"])
+    # The params to perform the query.
+    cd = {
+        "type": SEARCH_TYPES.RECAP,
+        "q": "chapter:7",
+        "court": " ".join(court_ids),
+        "case_name": "ready mix",
+        "filed_after": date(2020, 1, 1),
+    }
+
+    search_query = DocketDocument.search()
+    s, _ = build_es_base_query(search_query, cd)
+    s = s.extra(size=options["results_size"])
+    response = s.execute().to_dict()
+    extracted_data = [
+        {
+            "docket_absolute_url": item["_source"]["docket_absolute_url"],
+            "dateFiled": item["_source"]["dateFiled"],
+            "caseName": item["_source"]["caseName"],
+            "docketNumber": item["_source"]["docketNumber"],
+            "court_exact": item["_source"]["court_exact"],
+            "chapter": item["_source"]["chapter"],
+        }
+        for item in response["hits"]["hits"]
+    ]
+
+    # Save the json results file to: "ready_mix_cases/extracted_ready_mix_cases.json"
+    json_path = os.path.join(settings.MEDIA_ROOT, "ready_mix_cases")
+    if not os.path.exists(json_path):
+        os.makedirs(json_path)
+
+    json_file = os.path.join(
+        settings.MEDIA_ROOT,
+        "ready_mix_cases",
+        f"extracted_ready_mix_cases.json",
+    )
```

Comment:
You're an old man like me now, Alberto, but that's no excuse for using `os.path.join` when Pathlib exists. We grew up in the wrong era, but it's time to adapt to the ways. Put it in your brain: Whenever you think of `os.path`, `Pathlib` is probably the better tool.